lib/ClaimVerifier.ts

```diff
@@ -1,8 +1,6 @@
-import { OffchainClaim, DecodedToken } from './LoginStrategy.types';
-import * as jwt from 'jsonwebtoken';
+import { OffchainClaim } from './LoginStrategy.types';
 import { IRoleDefinition } from '@energyweb/credential-governance';
 import { IDIDDocument } from '@ew-did-registry/did-resolver-interface';
-import { ProofVerifier } from '@ew-did-registry/claims';
 
 export class ClaimVerifier {
```

Comment:
Even if this is not relates to this PR I wonder can ClaimVerifier be replaced by CredentialResolver and ClaimIssuerVerification from vc-verification now?

Reply:
I think so, I will create a ticket